src/encoded/static/components/__tests__/dbxref-test.js

```diff
@@ -561,6 +561,25 @@ describe('Test individual dbxref types', () => {
         });
     });
 
+    describe('Test DGGR', () => {
+        let dbxLinks;
+
+        beforeAll(() => {
+            const context = { '@type': ['FlyDonor'] };
+            const wrapper = mount(
+                <DbxrefList context={context} dbxrefs={['DGGR:109027', 'DGGR:101084']} />
+            );
+
+            dbxLinks = wrapper.find('a');
+        });
+
+        it('has the correct links', () => {
+            expect(dbxLinks.length).toBe(2);
```

Comment:
Testing for the length and testing for the correct values of links are two separate concerns. One succeeding those not mean the other one will. My recommendation would be to break them up to two unit test.

Reply:
Do you have a relevant citation to support that for React Jest tests?

I see Facebook tests for any number of things in one Jest test:
https://github.com/facebook/react/blob/f762b3abb1d951b366f3c8a2354aeedb7fbc32ff/packages/react/src/__tests__/ReactChildren-test.js#L813

Even Jest examples themselves do too:
https://github.com/facebook/jest/blob/master/examples/timer/__tests__/infinite_timer_game.test.js

Doesn't mean they’re right, but I think splitting up every function in this file needs some outside justification.

Reply:
TL;DR Answer: 
It is not big deal and seems to be a React-way of thinking about unit test. I will pass this.

Long Story Answer:
Consider these-

Look briefly at this question: https://stackoverflow.com/questions/2878717/multiple-asserts-in-a-unit-test

Look at answer: https://stackoverflow.com/a/2878912/178550

Note, Roy Osherove's "The Art of Unit Testing" is an older book that was groundbreaking in pushing unit testing. It is a much nicer read in my opinion than Ken Beck's pioneering and older TDD book which covered unit test. Anyway, assert is equivalent to Enzyme's "expect". 

Essentially, each unit test should test a concept- Ideally have only one assert (expect-test) or  a few that test a concept with a few expects. I should be able to look at a test run from Jest and each failed test should match one failed thing and not many. If it tells me many, I am getting into integration test territory. In this case, if the test fails, it could be that the length is wrong OR the content is wrong. Testing one thing or concept also is in support of Uncle Bob's (Robert Martin) single responsibility principle, which is part of his SOLID principles (https://en.wikipedia.org/wiki/SOLID). I do admit, following this idea may lead to having a lot more files and methods. I also do admit too this be a draconic principle fading away like React did  with having HTML in JavaScript (used to be an automatic code-review fail; not it is standard).